src/App.jsx

```diff
@@ -1,35 +1,32 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react";
+import "./App.css";
+import Header from "./Components/Header";
+import Navigation from "./Components/Navigation";
+import { Route, Routes } from "react-router-dom";
+import ArticleList from "./Components/ArticleList";
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [currentArticles, setCurrentArticles] = useState([]);
 
   return (
     <>
       <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
+        <Header />
+        <Navigation />
+        <Routes>
+          <Route
```

Comment:
You currently don't have anything on route "/" which impacts the user experience, you could have articles on "/" or set up a redirect. Also look to add some loading status to show a user that data is being loaded and that the site hasn't just crashed